src/common/image_cache.c

```diff
@@ -218,6 +222,13 @@ void dt_image_cache_write_release(dt_image_cache_t *cache, dt_image_t *img, dt_i
       struct dt_image_raw_parameters_t s;
       uint32_t u;
   } flip;
+  if(!img->aspect_ratio || img->aspect_ratio < .0001)
```

Comment:
This is equivalent to:
if(img->aspect_ratio==0 || img->aspect_ratio < .0001)
so to:
if(img->aspect_ratio < .0001)

Right?